src/SumOfRow.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by Yura on 01.05.2019.
+ */
+
+public class SumOfRow {
+    public static void main(String[] args) {
+        int num = 4;
+        int sum=0;
+        for(int i = 0; i <= num; i++){
+            int temp=1;
+            for(int j=1; j<=i; j++){
+                temp *= j;
```

Comment:
I dont think this code works properly. Check conditions again. (There should be minus somewhere)